website/content/docs/troubleshoot/troubleshoot-services.mdx

```diff
@@ -0,0 +1,136 @@
+  - To check that the right cluster is being dialed, run a DNS lookup for the upstream you are dialing. For example, run `dig backend.svc.consul` to return the IP address for the `backend` service. If the address you get from that is missing from the upstream IPs, it means that your proxy may be misconfigured.
+```
+
+1. Run the `consul troubleshoot proxy` command and specify the  Envoy ID or IP address with the `-upstream-ip` flag to identify the proxy you want to perform the troubleshooting process on. The following example uses the upstream IP to validate communication with the upstream service `backend`:
+  
+```shell-session
+  $ consul troubleshoot proxy -upstream-ip 10.4.6.160
+
+  ==> Validation
+   ✓ Certificates are valid
+   ✓ Envoy has 0 rejected configurations
+   ✓ Envoy has detected 0 connection failure(s)
+   ✓ Listener for upstream "backend" found
+   ✓ Route for upstream "backend" found
+   ✓ Cluster "backend.default.dc1.internal.e08fa6d6-e91e-dfe0-f6e1-ba097a828e31.consul" for upstream "backend" found
+   ✓ Healthy endpoints for cluster "backend.default.dc1.internal.e08fa6d6-e91e-dfe0-f6e1-ba097a828e31.consul" for upstream "backend" found
+   ✓ Cluster "backend2.default.dc1.internal.e08fa6d6-e91e-dfe0-f6e1-ba097a828e31.consul" for upstream "backend" found
+   ! No healthy endpoints for cluster "backend2.default.dc1.internal.e08fa6d6-e91e-dfe0-f6e1-ba097a828e31.consul" for upstream "backend" found
+     -> Check that your upstream service is healthy and running
+     -> Check that your upstream service is registered with Consul
+     -> Check that the upstream proxy is healthy and running
+     -> If you are explicitly configuring upstreams, ensure the name of the upstream is correct
+  ```
+
+In the example, troubleshooting upstream communication reveals that the `backend` service has two service instances running in datacenter `dc1`. One of the services is healthy, but Consul cannot detect healthy endpoints for the second service instance.
+
+The output from the troubleshooting process identifies service instances according to their [Consul DNS address](/consul/docs/discovery/dns#standard-lookup). Use the DNS information for failing services to diagnose the specific issues affecting the service instance.
+
+### Troubleshoot on Kubernetes
+
+To troubleshoot service-to-service communication issues in deployments that use Kubernetes, retrieve the upstream information for the pod that is experiencing communication failures and use the upstream information to identify the proxy you want to perform the troubleshooting process on.
+
+1. Run the `consul-k8s troubleshoot upstreams` command and specify the pod ID with the `-pod` flag to retrieve upstream information. Depending on your network’s configuration, the upstream information is either an Envoy ID or an IP address. The following example displays all transparent proxy upstreams in Consul service mesh from the given pod.
+
+```shell-session
```

Comment:
same thing with the rendering here: 

<img width="756" alt="image" src="https://user-images.githubusercontent.com/7556325/220872625-452a76b4-06d8-4295-b88d-a561ab881be6.png">

can you indent? also can you remove the new line between the command and output?